virttest/qemu_vm.py

```diff
@@ -1340,6 +1340,10 @@ def sort_key(dev):
             for pcic in params.objects("pci_controllers"):
                 dev = devices.pcic_by_params(pcic, params.object_params(pcic))
                 pcics.append(dev)
+            extra_root_port = params.get('extra_root_port', '0')
```

Comment:
there is no need to enter `str` as default value. The call of `int()` on integer succeeds as well...